src/Authenticator/AuthenticatorCollection.php

```diff
@@ -83,7 +83,7 @@ protected function _resolveClassName($class)
      * @return void
      * @throws \RuntimeException
      */
-    protected function _throwMissingClassError($class, $plugin)
+    protected function _throwMissingClassError($class, $plugin): void
```

Comment:
No typehint for arguments?

Reply:
for typehints I will open another PR
just `:void` for return type was required by core here :)

Reply:
looks I have to do it in this PR , php 7.1 needs it